projects/vdk-core/src/vdk/internal/builtin_plugins/config/log_config.py

```diff
@@ -204,6 +204,54 @@ def ensure_logging_for_current_test() -> None:
         configure_loggers(fname, str(time()))
 
 
+class VdkBoundLogger(logging.LoggerAdapter):
```

Comment:
Move this to a separate file/module. 

log_config module is already doing too many things. It should be refactored to remove things and not add new things. 

But also VdkBoundLogger is a public (plugin-facing) interface so it's going ot be reused a lot of times. You want it in a file with only its own imports otherwise users (modules) importing VDKBoundLogger ot use it will import bunch of unnecessary imports for them


Also the "config" builtin_plugin doesn't seem the best place. 

Overall we have a definition for https://github.com/vmware/versatile-data-kit/wiki/Coding-Standard#public-python-interfaces here. 

Last, plugins have been really allowed to use anything under `vdk.internal.core` or `vdk.interna.plugin` but generally discouraged to use things under vdk.internal.builtin_plugins namespaces

So best place would be in vdk.internal.core.logging.py module

Reply:
gotcha